src/theme.ts

```diff
@@ -252,4 +252,36 @@ export default {
 		extend: 'border: none;',
 		color: colors.primary.main,
 	},
+	checkBox: {
+		size: '20px',
+		color: colors.primary.main,
+		border: {
+			color: {
+				dark: colors.quartenary.dark,
+				light: colors.quartenary.dark,
+			},
+			width: '1px',
+		},
+		check: {
+			radius: '4px',
+			thickness: '2px',
+		},
+		hover: {
+			border: {
+				color: {
+					dark: colors.quartenary.dark,
+					light: colors.quartenary.dark,
+				},
+			},
+		},
+		toggle: {
```

Comment:
I am not a huge fan of the slow color animation on hover, if anything at least it can be made faster IMO. I am fine as it is though, just an opinion.